pkg/kademlia/kademlia.go

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// Package kademlia provides the implementation for the
+// Swarm forwarding-kademlia. Kademlia makes sure you're
```

Comment:
Technically, forwarding Kademlia is implemented by this package, but also by the retrieval protocol. I would say:
`Package Kademlia makes sure that you are connect to the right peers at any point in time, based on Kademlia topology. It forms the basis of Swarms forwarding Kademlia`

Reply:
I realised that kademlia package already had a description. removing this...